blackbox/docs/sql/information_schema.txt

```diff
@@ -643,3 +644,47 @@ CrateDB based to the current SQL standard (see :ref:`sql_supported_features`)::
 :comments:
     Either ``NULL`` or shows a comment about the supported status of the
     feature
+
+.. _information_schema_ingest:
+
+Ingestion Rules
+===============
+
+The ingestion rules table contains rules created by ``CREATE INGEST RULE``
+statements (see :ref:`create-ingest-rule`)::
+
+    cr> select rule_name, source_ident
+    ... from information_schema.ingestion_rules
+    ... order by rule_name asc limit 5;
+     +--------------+------------------+
+     | rule_name    | source_ident     |
+     +--------------+------------------+
+     | v4           | mqtt             |
+     +--------------+------------------+
+     SELECT 1 row in set (... sec)
+
+Columns
+-------
+
++--------------------+-------------+
+| Name               | Data Type   |
++====================+=============+
+| rule_name          | String      |
++--------------------+-------------+
+| source_ident       | String      |
++--------------------+-------------+
+| target_table       | String      |
++--------------------+-------------+
+| condition          | String      |
++--------------------+-------------+
+
+
+:rule_name:
+    The identifier of the ingestion rule.
+:source_ident:
+    The identifier of the ingestion source (eg. ``mqtt``).
+:target_table:
+    The identifier of the target table.
+:condition:
+    A boolean expression by which messages to the ingestion source
```

Comment:
I think this description is a bit confusing. How about "A boolean expression used to filter the source data" ?